src/Java.Interop/Java.Interop/JniTypeSignature.cs

```diff
@@ -172,7 +172,7 @@ public static bool TryParse (string signature, [NotNullWhen (true)] out JniTypeS
 
 		public override int GetHashCode ()
 		{
-#if NETCOREAPP
+#if NET5_0
```

Comment:
It might be better to leave this as it was, or use `NET`, as this wouldn't work anymore if it was moved to .NET 6.

Reply:
I think these defines were changed so that NET5_0 will be defined in .NET 6 as well.

Reply:
I think these defines were changed so that NET5_0 will be defined in .NET 6 as well.

Reply:
They keep changing their mind on this but I don’t this this will be defined in 6.0, but these will:

https://github.com/dotnet/designs/pull/165

Reply:
The mentioned design still shows NET5_0 define for net6.0. It looks like it is still evolving so it might be indeed safer to keep `NETCOREAPP` around meanwhile.